registry/storage/driver/swift/swift_test.go

```diff
@@ -149,4 +148,24 @@ func TestEmptyRootList(t *testing.T) {
 			t.Fatalf("unexpected string in path: %q != %q", path, storagedriver.PathRegexp)
 		}
 	}
+
+	// Create an object with a path nested under the existing object
+	err = rootedDriver.PutContent(ctx, filename+"/file1", contents)
```

Comment:
See my comment in the commit message about how this won't work for all storage drivers. Any thoughts?

> On Nov 9, 2015, at 19:04, Stephen Day <notifications@github.com> wrote:
> 
> In registry/storage/driver/swift/swift_test.go:
> 
> > @@ -149,4 +148,24 @@ func TestEmptyRootList(t *testing.T) {
> >  			t.Fatalf("unexpected string in path: %q != %q", path, storagedriver.PathRegexp)
> >  		}
> >  	}
> > +
> > +	// Create an object with a path nested under the existing object
> > +	err = rootedDriver.PutContent(ctx, filename+"/file1", contents)
> This test seems like it should belong in the primary test suite.
> 
> —
> Reply to this email directly or view it on GitHub.
>